src/lib.rs

```diff
@@ -1280,7 +1280,7 @@ impl ExpressionStatus {
     }
 }
 
-mod errors {
+pub mod errors {
```

Comment:
Err, no. It's required. 😁 

`pub mod` is for exposing a mod to external crates.
`pub use` is to re-export a function inside another module. This allows you to present an external interface that may not directly map to your internal code organization.
([From the book](https://doc.rust-lang.org/book/crates-and-modules.html#re-exporting-with-pub-use)).

To make sure, I've just tested it again.

Doesn't work with the master in my Cargo.toml:
```
duct = { git = "https://github.com/oconnor663/duct.rs.git" }
```

Error:
```
error: module `errors` is private
 --> src/errors.rs:6:20
  |
6 |         ShellError(duct::errors::Error, duct::errors::ErrorKind);
  |                    ^^^^^^^^^^^^^^^^^^^

error: module `errors` is private
 --> src/errors.rs:6:20
  |
6 |         ShellError(duct::errors::Error, duct::errors::ErrorKind);
  |                    ^^^^^^^^^^^^^^^^^^^

error: module `errors` is private
 --> src/errors.rs:6:41
  |
6 |         ShellError(duct::errors::Error, duct::errors::ErrorKind);
  |                                         ^^^^^^^^^^^^^^^^^^^^^^^

error: module `errors` is private
 --> src/errors.rs:6:41
  |
6 |         ShellError(duct::errors::Error, duct::errors::ErrorKind);
  |                                         ^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 4 previous errors
```

Works with the PR branch in my Cargo.toml:
```
duct = { git = "https://github.com/mre/duct.rs.git", branch = "patch-1" }
```

So, I guess we should still merge this. We could replace `pub use errors::*;` with `use errors::*;` though, as it's imported in the crate root.